test/acceptance-tests/use-cases/test-scripts/create-from-templates.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const app = require('../../../../');
+const expect = require('../../../helpers/expect');
+const fs = require('fs-extra');
+const loopback = require('loopback');
+const path = require('path');
+const testSupport = require('../../../helpers/test-support');
+const util = require('util');
+const workspaceManager = require('../../../../component/workspace-manager');
+
+const Workspace = app.models.Workspace;
+app.on('booted', function() {
+  app.emit('ready');
+});
+
+module.exports = function() {
+  const testsuite = this;
+  this.Given(/^that the templates are loaded$/, function(next) {
+    testsuite.destinationPath =
+      path.resolve(__dirname, '../../../', 'sandbox2');
```

Comment:
I'm curious as to why we need sandbox2 -- can't we just wipe the original sandbox before executing your tests? (ie. empty out the original sandbox dir during the setup phase).